.github/ISSUE_TEMPLATE/infrastructure.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Infrastructure Issue
+about: Report an issue with Open Robotics infrastructure
+
+---
+
+- [ ] I have checked [ROS Answers](https://answers.ros.org) for similar reports and solutions.
+- [ ] I have checked the [ROS Status page](https://status.ros.org) and my issue is not listed there.
+- [ ] I have checked the [open issues](https://github.com/osrf/infrastructure/issues) for similar reports.
+
+## Problem description
+
+Example: When running `apt update` the following output appears
+
+```
+W: GPG error: http://packages.ros.org/ros/ubuntu focal InRelease: The following signatures were invalid: EXPKEYSIG F42ED6FBAB17C654 Open Robotics <info@osrfoundation.org>
+E: The repository 'http://packages.ros.org/ros/ubuntu focal InRelease' is not signed.
+```
+
+### Security issues
```

Comment:
I moved this up into the checklist and linked to the SECURITY.md this should allow us to avoid this content here in the bottom just getting duplicated into every ticket as unnecessary boilerplate instructions.